layouts/compliance/pci-dss.html

```diff
@@ -0,0 +1,225 @@
+
+                <li>
+                    Ensure that only authorized personnel can access sensitive information.
+                </li>
+            </ul>
+            <h4>8. Identify and Authenticate Access to System Components</h4>
+
+            <ul>
+                <li>
+                    Implement strong authentication methods for accessing your applications.
+                </li>
+
+                <li>Enforce password complexity and expiration policies.</li>
+            </ul>
+            <h4>9. Restrict Physical Access to Cardholder Data</h4>
+            <ul>
+                <li>
+                    Ensure that AWS physical security controls are adequate for the infrastructure
+                    hosted in AWS.
+                </li>
+
+                <li>
+                    Review AWS's compliance with physical security requirements as part of
+                    their shared responsibility model.
+                </li>
+            </ul>
+            <h4>
+                10. Track and Monitor All Access to Network Resources and Cardholder Data
+            </h4>
+            <ul>
+                <li>
+                    Enable CloudTrail and CloudWatch to log and monitor access to your AWS
+                    resources.
+                </li>
+
+                <li>
+                    Implement logging for your web app to track access to cardholder data.
+                </li>
+
+                <li>Regularly review logs and set up alerts for suspicious activity.</li>
+            </ul>
+            <h4>11. Regularly Test Security Systems and Processes</h4>
+            <ul>
+                <li>
+                    Conduct regular vulnerability scans and penetration testing on your
+                    hosted applications.
+                </li>
+
+                <li>
+                    Implement continuous monitoring and testing of your security systems.
+                </li>
+            </ul>
+            <h4>
+                12. Maintain a Policy That Addresses Information Security for All Personnel
+            </h4>
+            <ul>
+                <li>Develop and enforce an information security policy.</li>
+
+                <li>
+                    Train your staff on security practices and the importance of protecting
+                    cardholder data.
+                </li>
+            </ul>
+            <h4>Additional Considerations:</h4>
+            <ul>
+                <li>
+                    <strong>Segmentation:</strong> Ensure that you properly segment your
+                    environment to isolate cardholder data from other non-sensitive areas.
+                </li>
+
+                <li>
+                    <strong>Compliance Documentation:</strong> Maintain records of all
+                    compliance efforts, security measures, and audits for validation and
+                    reporting purposes.
+                </li>
+            </ul>
+        </div>
```

Comment:
I don't know where but we should have a section that lists out all the clickable links to each of the services